static/js/publisher/release/components/releasesTable/cellViews.js

```diff
@@ -1,3 +1,4 @@
+/* eslint-disable */
```

Comment:
:eyes:

Reply:
Yeah that's right. We will need it in the next couple of tasks so didn't want to get rid of it entirely